cli/src/commonMain/kotlin/StartCommand.kt

```diff
@@ -80,7 +81,8 @@ class StartCommand : SuspendingCliktCommand(name = "start") {
             "--parameters-file",
             envvar = "OSC_RUNS_START_PARAMETERS_FILE",
             help = "The path to a JSON file containing the run configuration " +
-                    "(see https://eclipse-apoapsis.github.io/ort-server/api/post-ort-run)."
+                    "(see https://eclipse-apoapsis.github.io/ort-server/api/post-ort-run).",
+            completionCandidates = CompletionCandidates.Path
```

Comment:
Could you briefly explain in the commit message why it was not working before, and what setting `completionCandidates` changes to fix this?